docs/overview.md

```diff
@@ -65,27 +65,27 @@ When sending test reports to Apiary, Dredd inspects the environment where it was
 - hostname (string) - `DREDD_HOSTNAME` or hostname of the OS
 - CI (boolean) - looks for `TRAVIS`, `CIRCLE`, `CI`, `DRONE`, `BUILD_ID`, ...
 
-## Testing API Documentation
+## Testing API description
 
 ### Documentation Testability
 
-API Blueprint allows usage of URI templates. If you want to have API documentation to be complete and testable, do not forget to describe all URI-used parameters and provide examples to make Dredd able to expand URI templates with given example values.
+API Blueprint allows usage of URI templates. If you want to have API description to be complete and testable, do not forget to describe all URI-used parameters and provide examples to make Dredd able to expand URI templates with given example values.
 
 ### Isolation
 
 API Blueprint structure conforms to the REST paradigm, so in the API Blueprint are documented Resources and their Actions.
```

Comment:
Yup, some places are still left intact. E.g. here the sentence says "API Blueprint structure conforms to the REST paradigm", which isn't valid to OAI in my opinion. Rephrasing such sections should be part of "Let's add Swagger docs" PR.